src/main/java/mate/academy/spring/service/mapper/CinemaHallDtoMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.model.CinemaHall;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallDtoMapper {
+    public CinemaHall fromReqDto(CinemaHallRequestDto reqDto) {
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setCapacity(reqDto.getCapacity());
+        cinemaHall.setDescription(reqDto.getDescription());
+        return cinemaHall;
+    }
+
+    public CinemaHallResponseDto toRespDto(CinemaHall cinemaHall) {
```

Comment:
according to checklist, same for others
```suggestion
    public CinemaHall toModel(CinemaHallRequestDto reqDto) {
        CinemaHall cinemaHall = new CinemaHall();
        cinemaHall.setCapacity(reqDto.getCapacity());
        cinemaHall.setDescription(reqDto.getDescription());
        return cinemaHall;
    }

    public CinemaHallResponseDto toDto(CinemaHall cinemaHall) {
```